docs/contributors/release.md

```diff
@@ -186,25 +186,55 @@ If you don't have access to [make.wordpress.org/core](https://make.wordpress.org
 
 ## Packages Releases and WordPress Core Updates
 
-WordPress Core Updates are based on the `g-minor` branch. Releasing packages in order to update WordPress Core involves updating the `g-minor` branch (the workflow depends on whether it's a minor or major WordPress release) and run the package release process.
+The Gutenberg repository mirrors the WordPress SVN repository in terms of branching for each SVN branch, a corresponding Gutenberg `wp/*` branch is created:
 
-### Major WordPress Releases
+ - The `wp/trunk` branch contains all the packages that are published and used in the `trunk` branch of WordPress.
+ - A Gutenberg branch targetting a specific WordPress release and its minors is created (example `wp/5.2`) based on the `wp/trunk` Gutenberg branch when the WordPress `trunk` branch is marked as "feature-freezed". (This usually happens when the first `beta` of the next WordPress major version is released).
```

Comment:
Currently the https://github.com/wordpress/wordpress-develop repo uses `x.y` for branches, and `x.y.z` for tags, this includes the major releases tagged as `x.y.z`, for example `5.1.0`

• https://github.com/WordPress/wordpress-develop/branches 
• https://github.com/WordPress/wordpress-develop/tags

I suggest rather than `wp/5.2` that `wp/5.2.0` is used so that Gutenberg uses a consistent 3 digit nomenclature?

Reply:
I think Gutenberg should use the same thing `wp/5.2` for the branch and `x.y.z` for tags. the  `wp/5.2` will serve for all `5.2.1`, `5.2.2`...

Reply:
When WordPress releases a major release it is both _branched_ and _tagged_, so a `5.0` branch is created, and a `5.1.0` tag is created:

• `5.0` branch https://github.com/WordPress/wordpress-develop/tree/5.1
• `5.1.0` tag  https://github.com/WordPress/wordpress-develop/tree/5.1.0

All `package.json` files use the 3 digit nomenclature:

https://github.com/WordPress/wordpress-develop/blob/5.1.0/package.json#L3:
> `"version": "5.1.0",`

Also, to note that the `5.0` branch has now had 4 minor releases so the _5.0 branch_ `package.json` file is now `5.0.4`:
https://github.com/WordPress/wordpress-develop/blob/5.0/package.json#L3:
> `"version": "5.1.0",`

----

If as you suggest, Gutenberg *only* use `wp/5.2` for all `5.2.x` releases I think that would work and be less work to maintain....

Reply:
Tags are automatically created by Lerna when we publish the packages. we can probably also tag `wp-5.2.1` ourselves, but the issue is that at the time of the npm release, we don't know yet if it's the final npm release that will be merged in this WP version and if we won't need more. So maybe just using the branch and the lerna tags is enough.